src/psyclone/psyir/nodes/operation.py

```diff
@@ -197,7 +197,9 @@ class BinaryOperation(Operation):
         # Other Maths Operators
         'SIGN', 'MIN', 'MAX',
         # Query Operators
-        'SIZE', 'LBOUND', 'UBOUND'
+        'SIZE', 'LBOUND', 'UBOUND',
+        # Matrix and Vector Operators
+        'MATMUL'
```

Comment:
Please document MATMUL in the docstring below.

Reply:
Done. Hopefully in a good enough way.